app/controllers/webhooks_controller.rb

```diff
@@ -125,7 +125,17 @@ def handle_payment_intent_succeeded(
       if seller_id.eql?(Seller::POOL_DONATION_SELLER_ID)
         PoolDonationValidator.call({ type: item['item_type'] })
 
-        @donation_sellers = Seller.filter_by_accepts_donations
+        # TODO(jtmckibb): This is a very inefficient sort, since each time we
+        #                 call amount_raised, it has to fetch all of the
+        #                 associated Items, DonationDetails, and GiftCardDetails
+        #                 Then for each GiftCardDetail, it has to fetch every
+        #                 amount in an N+1 query, then sum everything.
+        #                 This all in an N log N sort is just a bad. Ideally,
+        #                 we would memoize amount_raised, and fix the N+1 query
+        #                 in GiftCardDetail that calculates amount.
```

Comment:
I want to fix this in a follow up since it doesn't *really* matter too much being that we only have 6 sellers. The real ideal would to be able to store this as a calculated column somehow and index on it